src/components/questionary/Questionary.tsx

```diff
@@ -28,7 +28,7 @@ function Questionary({
   handleReset,
   displayElementFactory,
 }: QuestionaryProps) {
-  const isTabletOrMobile = useMediaQuery('(max-width: 1224px)');
+  const isTabletOrMobile = useMediaQuery('(max-width: 500px)');
```

Comment:
Like this maybe it should be called only `isMobile` instead of `isTabletOrMobile` because this resolution lower than 500 is not for tablets.